### PR TITLE
fix: sentry import

### PR DESCRIPTION
**Pull request by siiick**
Fixes https://github.com/pascaliske/ngx-sentry/issues/18

### CODE REVIEW

File: projects/ngx-sentry/src/lib/sentry.interceptor.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Injectable, Inject } from '@angular/core'
 import { HttpInterceptor, HttpRequest, HttpHandler, HttpErrorResponse } from '@angular/common/http'
-import { captureMessage } from '@sentry/browser/esm'
+import * as Sentry from '@sentry/browser'
```
Comment (pascaliske) [inline projects/ngx-sentry/src/lib/sentry.interceptor.ts]: Please stick with the destructuring imports `{ ... }` as they're more expressive which functions are used in this file.

**Comment by pascaliske** [comment]
Hello @siiick,

Thanks for your PR! 🙂

The imports from `/esm` were intentionally at first to reduce the library size but I think it would be better to move sentry to the peerDependencies instead. This would allow every consumer to choose between `@sentry/browser` and `@sentry/browser/esm`.

Therefore could you please change all imports of sentry in this library to `@sentry/browser` to be consistent. Afterwards I can merge this. 🙂

**Comment by siiick** [comment]
Hello @pascaliske,

I agree `@sentry/browser` should be a peer dependecy.
All sentry imports are now consistent :)

File: projects/ngx-sentry/src/lib/sentry.interceptor.ts (hunk 1 above)
Comment (siiick) [reply projects/ngx-sentry/src/lib/sentry.interceptor.ts]: ok done

**Comment by pascaliske** [comment]
Merged - Will do a release in the near future. 🙂

**Pull request merged**